meta/vigotech.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "vigotech",
+  "author": "Pedro Figueras www.pedrofigueras.com",
+  "license": "CC BY-NC-ND 4.0",
+  "vector": "http://hexb.in/vector/vigotech.svg",
+  "description": "Tech User Group Alliance",
```

Comment:
Please adjust to `"Tech user group alliance from Vigo, Spain"`

Reply:
Done!